### PR TITLE
Add OpenTracing example and fix other examples

### LINKED ISSUE #7: Add Opentracing Shim example
Add an example that uses the Opentracing shim.

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #7

### CODE REVIEW

File: docker-compose.yml.bak [hunk 1]
```diff
@@ -0,0 +1,396 @@
+version: "3.7"
```
Comment (codeboten) [inline docker-compose.yml.bak:1]: this file should probably be removed

File: docker-compose.yml [hunk 2]
```diff
@@ -1,301 +1,37 @@
 version: "3.7"
 services:
```
Comment (codeboten) [inline docker-compose.yml:2]: looks like this was removed for local testing?

File: python/opentracing/client.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+trace.set_tracer_provider(TracerProvider())
+configure_opentelemetry()
+otel_tracer = trace.get_tracer(__name__)
+shim = create_tracer(trace.get_tracer_provider())
+
+
+def send_requests(target):
+    integrations = ["pymongo", "redis", "sqlalchemy"]
+    # with global_tracer().start_active_span("client operation"):
+    with shim.start_active_span("client operation"):
```
Comment (codeboten) [inline python/opentracing/client.py:29]: is it possible to use the global_tracer here?

File: python/opentracing/Dockerfile.client [hunk 4]
```diff
@@ -0,0 +1,12 @@
+FROM python:3.7
+RUN pip install flask pymongo redis requests sqlalchemy
+RUN pip install lightstep
+RUN pip install opentelemetry-launcher
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-api
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-sdk
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=opentelemetry-instrumentation
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=instrumentation/opentelemetry-instrumentation-opentracing-shim
+RUN pip install git+https://github.com/ocelotl/opentelemetry-python.git@issue_305#subdirectory=instrumentation/opentelemetry-instrumentation-requests
```
Comment (codeboten) [inline python/opentracing/Dockerfile.client:9]: Any reason these are being installed from a fork of otel?

Comment (ocelotl) [reply python/opentracing/Dockerfile.client:9]: I still have pending changes that are in open PRs, not yet merged into master. I tried using branches from the main `opentelemetry-python` repo, but they were not present there.

File: docker-compose.yml.bak (hunk 1 above)
Comment (ocelotl) [reply docker-compose.yml.bak]: Fixed

File: docker-compose.yml (hunk 2 above)
Comment (ocelotl) [reply docker-compose.yml]: Fixed

File: python/opentracing/client.py (hunk 3 above)
Comment (ocelotl) [reply python/opentracing/client.py]: Fixed

**Pull request merged**